File: 1.1.cpp
#include <iostream>
#include <fstream>
#include <sstream>
#include <string>
#include <vector>

int main()
{
    std::string line;
    std::vector<std::string> DataArray;
    std::vector<int> NumArrary;

    std::ifstream myfile("input1.txt");

    if (!myfile) //Always test the file open.
    {
        std::cout << "Error opening output file" << std::endl;
        system("pause");
        return -1;
    }
    while (std::getline(myfile, line))
    {
        DataArray.push_back(line);
    }
    for(int i = 0; i < DataArray.size(); i++)
    {
        std::stringstream word(DataArray[i]);
        int temp = 0;
        word >> temp;
        NumArrary.push_back(temp);
    }
    int prev = NumArrary[0];
    int howMany = 0;
    for(int i = 1; i < NumArrary.size(); i++)
    {
        if(NumArrary[i] > prev ){
            howMany++;
        }
        prev = NumArrary[i];
    }
        std::cout <<howMany << std::endl;
}

File: 2022/Day1/1.1.py
def main():
       f = open("input1.txt","r")
       Lines = f.readlines()
       bestSoFar = 0
       comparisor = 0
       for line in Lines:
            if(line.strip()):
                comparisor += int(line)
            else:
                if(comparisor > bestSoFar):
                    bestSoFar = comparisor
                comparisor = 0
       print(bestSoFar)

if __name__ == "__main__":
    main()